frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -449,7 +464,7 @@ private void prepareStandardView(RenderRequest request) throws IOException {
             } else {
                 projectList = projectClient.refineSearch(searchtext, filterMap, user);
             }
-            for(Project project:projectList){
+            for (Project project : projectList) {
```

Comment:
[hyperbole] I hope this avid love for spaces will not turn someday into reordering of methods alphabetically throughout existing codebase.[/hyperbole] I mean, guys, let's not make changes for changes' sake. We don't have a consistent code style even within this PR (look at the spaces before opening curly braces, for example). I suggest to not change code style unless you can argue that it looks absolutely ugly.